### DISCUSSION TITLE
初始化完成后新增配置项未生效

### DISCUSSION THREAD

**Discussion by llxlr** [nonebot/discussions#150] [问答] [answered]
使用 `nonebot 2.0.0-beta.1` 时，在初始化完成后新增配置项

```python
config = nonebot.get_driver().config
config.custom_config = {"hello": "world"}
```

![Snipaste_2022-05-22_04-37-47](https://user-images.githubusercontent.com/33483752/169668257-ac407e68-1492-4205-a9eb-abf1f5e2e80b.png)

如图所见，`Loaded Config` 中未出现 `custom_config` ，使用 `nonebot 2.0.0a16` 时是正常的。

**Comment by mnixry** [comment, marked as answer]
这个日志似乎是初始化的时候打的，所以没有
你要是去取的话应该还是有的？

---

虽然但是建议把所有配置写进.env，不建议动态修改config

**Comment by yanyongyu** [comment]
这个日志是初始化时log的，如果你在初始化之后修改自然是不会显示的
